gpt_register/register.py

```
@@ -251,6 +251,33 @@ def run(proxy: Optional[str]) -> tuple:
+                                "referer": "https://auth.openai.com/email-verification",
+                                "accept": "application/json",
+                                "content-type": "application/json",
+                                "openai-sentinel-token": sentinel,
+                            },
+                            json_body={},
+                            proxies=proxies,
+                            timeout=15,
+                            retries=1,
+                        )
+                        print(f"[*] OTP 后续步骤状态: {otp_next_resp.status_code}")
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)